prime-router/metadata/organizations-test.yml

```diff
@@ -165,4 +165,38 @@
           redox_destination_name: "CDC Montgomery County PDH Destination (s)"
           redox_source_id: d89d4057-930f-4c5b-bb39-8cddb326e928
           redox_source_name: "Prime Data Hub (Staging)"
-        format: REDOX
\ No newline at end of file
+        format: REDOX
+
+  - name: pa-chester-phd
+    description: Health Department - Chester County, Pennsylvania
+    services:
+      - name: elr-chester-download-test
+        topic: covid-19
+        schema: strac/strac-covid-19
+        jurisdictionalFilter: [ "matches(ordering_facility_state, PA)", "matches(ordering_facility_county, Chester, Delaware)" ]
+        transforms: { deidentify: false }
```

Comment:
So, PA doesn't care about patient's address location, only the testing facility?   That was the use case for `filterByCounty` which is `true` for either patient **or** facility in the right state/county.